src/richie/apps/courses/settings/__init__.py

```diff
@@ -246,6 +246,16 @@ def richie_placeholder_conf(name):
         "name": _("Assessment and Certification"),
         "plugins": ["CKEditorPlugin"],
     },
+    "courses/cms/course_detail.html course_required_equipment": {
+        "name": _("Required equipment"),
+        "plugins": ["PlainTextPlugin"],
+        "limits": {"PlainTextPlugin": 1},
+    },
+    "courses/cms/course_detail.html course_accessibility": {
+        "name": _("Accessibility"),
+        "plugins": ["PlainTextPlugin"],
+        "limits": {"PlainTextPlugin": 1},
+    },
```

Comment:
After some local tests, I realized you should use a `CKEditorPlugin` but with a basuc configuration. To enable the basic configuration, you have to append new placeholders here : https://github.com/openfun/richie/blob/737f76685d5237ef681d471325bc1dc54daf5c3a/src/richie/apps/courses/settings/__init__.py#L531-L534